examples/applications/plot_face_recognition.py

```diff
@@ -45,9 +29,9 @@
 logging.basicConfig(level=logging.INFO, format="%(asctime)s %(message)s")
 
 
-# #############################################################################
+# %%
 # Download the data, if not already on disk and load it as numpy arrays
-
+# %%
```

Comment:
We will need to keep the sphinx-gallery style that is slightly different from the vs code style

```suggestion
```